docs/agents/sematext-agent/releasenotes.md

```diff
@@ -39,6 +39,20 @@ NULL
 
 -->
 
+## Version 4.3.1
+
+Date: May 15, 2024
+
+### Improvements
+
+- Support for monitoring services running on the Containerd runtime in Kubernetes has been added.
+- Implemented a new billing system for CronJobs and Jobs that is more favorable to customers than regular container billing.
+- Enhanced the reliability of the Kubernetes initialization structure.
+
+### Bug Fixes
+
+- Resolved a service discovery edge-case bug occurring during cold starts in ElasticSearch for Kubernetes and Docker environments.
```

Comment:
Let's remove this one and mention only ESv8.
```
- Resolved a service discovery bug occurring with ElasticSearch v8.
```

Reply:
I actually copied what you wrote in the previous release just to be explicit about the new envs, but OK, modifying